components/Heroes/BenchmarkBadge.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Grid, Row, Col } from 'react-flexbox-grid';
```

Comment:
yea, I mean that library is just using media queries under the hood. We use flexbox, but some displays of data just won't 'squash down' properly or overflow to the next line properly. I'm cool with using this, I just don't like adding lots of markup to the component vs. just adding media queries in css.